XamFormsSample/XamFormsSample/XamFormsSample.csproj

```diff
@@ -11,6 +11,11 @@
   </ItemGroup>
   
   <ItemGroup>
+    <None Remove="MainPage.xaml" />
+    <None Remove="App.xaml" />
```

Comment:
I think it's unfortunate that this is required in VSfM. There should be no harm in having "None" items in the tree.

I can accept the merge for now as a known issue, but I would strongly urge this to be fixed to match VS.

To fix though, can you please update this to be Remove="**\*.xaml" to at least have the minimum amount of noise and also add a comment pointing to the Bugzilla issue necessitating it?

Thanks!